test/spec/modules/prebidServerBidAdapter_spec.js

```diff
@@ -475,5 +475,62 @@ describe('S2S Adapter', () => {
       config.setConfig({ s2sConfig: options });
       sinon.assert.calledOnce(logErrorSpy);
     });
+
+    it('should log error when vendor does not exist', () => {
```

Comment:
Is there a test to show there's a failure when 'defaultVendor' doesn't exist but endpoint isn't specified?  This will show there's no default-defaultVendor.

Reply:
Hey @bretg - I'm not sure I follow what you mean.  Can you please clarify further when you get the chance?

Reply:
I think he's saying to add a test case to assert an error when both endpoint & vendor default are not specified in the S2S config (or vendor default doesn't exist)

Reply:
OK, thanks for that clarification.  I've added a test case in the next commit.